client/src/rpc/impls/common.rs

```diff
@@ -108,7 +108,10 @@ impl RpcImpl {
             .downcast_ref::<ConsensusGraph>()
             .expect("downcast should succeed");
         info!("RPC Request: cfx_gasPrice()");
-        Ok(consensus_graph.gas_price().unwrap_or(0.into()).into())
+        Ok(consensus_graph
+            .gas_price()
+            .unwrap_or(1000000000.into())
```

Comment:
better define a const similar to 
pub const CONFLUX_TOKEN: u64 = 1_000_000_000_000_000_000;